src/toil_scripts/gatk_concordance/gatk_concordance.py

```
@@ -0,0 +1,103 @@
+            except subprocess.CalledProcessError as cpe:
+                raise RuntimeError(
+                    '\nNecessary file could not be acquired: %s. Got error "%s". Check input URL' % (url, cpe))
+            except OSError:
+                raise RuntimeError('Failed to find "curl". Install via "apt-get install curl"')
+    assert os.path.exists(file_path)
+    return job.fileStore.writeGlobalFile(file_path)
+
+
+# copied from germline.py
+def read_from_filestore_c(job, work_dir, ids, *filenames):
+    for filename in filenames:
+        if not os.path.exists(os.path.join(work_dir, filename)):
+            job.fileStore.readGlobalFile(ids[filename], os.path.join(work_dir, filename))
+
+
+def batch_start(job, input_args):
+    shared_files = ['ref.fa', 'ref.fa.fai', 'eval.vcf', 'comp.vcf']
+    shared_ids = {}
+    for file_name in shared_files:
+        url = input_args[file_name]
+        shared_ids[file_name] = job.addChildJobFn(download_url_c, url, file_name).rv()
+    job.addFollowOnJobFn(concordance, shared_ids, input_args)
+    
+
+def concordance(job, shared_ids, input_args):
+    """
+    Evaluate concordance between two VCFs with GATK.
+    """
+    work_dir = job.fileStore.getLocalTempDir()
+    input_files = ['ref.fa', 'ref.fa.fai', 'eval.vcf', 'comp.vcf']
+    read_from_filestore_c(job, work_dir, shared_ids, *input_files)
+
+    eval_name = input_args['eval.vcf'].split('/')[-1]
+    comp_name = input_args['comp.vcf'].split('/')[-1]
+    output = '%s_vs_%s.concordance' % (eval_name, comp_name)
+    command = ['-T', 'GenotypeConcordance',
+               '-R', 'ref.fa',
+               '--eval', 'eval.vcf',
+               '--comp', 'comp.vcf',
+               '-o', output
+    ]
+    try:
+        docker_call(work_dir = work_dir,
+                    tool_parameters = command,
+                    tool = 'quay.io/ucsc_cgl/gatk',
+                    sudo = input_args['sudo'])
+    except:
```

Since this is a pretty simple pipeline and there aren't variable arguments to the concordance tool, I would drop the `try` and just let the job fail. `docker_call` reports the full command line in the log.